Show error snackbar when access token expired

rasmustautsglia opened the pull request:
**Jira issue:**
https://glia.atlassian.net/browse/MOB-3261

**What was solved?**
When a visitor's access token is expired, CoreSDK sends a specific error message to WidgetSDK, which is then displayed on a snack bar.

**Release notes:**

 - [ ] Feature
 - [X] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

- [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [X] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from iOS SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3589734507/Logging+from+iOS+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**

igorkravchenko reviewed (commented):
Since this PR adds new log messages, it has to be communicated to Android team if corresponding log messages have to be added for their platform as well as suggested by PR template. Currently this is not reflected in **Additional Info** section of this PR.

rasmustautsglia commented:
> Since this PR adds new log messages, it has to be communicated to the Android team if corresponding log messages have to be added for their platform as well as suggested by the PR template. Currently, this is not reflected in the Additional Info section of this PR.

After communicating with @andrews-moc we agreed that this log can stay, even though there is no counterpart on the Android side. I have updated the PR description by mentioning new logs under Additional Info

yurii-glia commented:
why this PR targeted to master instead of development?

rasmustautsglia commented:
I rebased the branch on development yesterday and force pushed. Perhaps that changed it.

> Base automatically changed from development to master [18 hours ago](https://github.com/salemove/ios-sdk-widgets/pull/938#event-12818771507)

I reverted it now
